src/test-run/index.js

```diff
@@ -954,6 +952,9 @@ ServiceMessages[CLIENT_MESSAGES.ready] = function (msg) {
 
     this._clearPendingRequest();
 
+    if (this.disconnected)
```

Comment:
I guess it better to move the error creation right after `this.debugLog.driverMessage(msg);` code line.